book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './person'
+require_relative './rental'
+
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] Kindly consider adding rentals to the attr_accessor so it can be accessible